CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Provide flexibility to for Store properties to be set via the Environment.
 - Changing the way Resource and ResourceProperty items are represented in dialogs. We include prefixes to make their identification easier.
 - Protect initServerProviders() method
+- Migrated from Javax to Jakarta packages, including upgrade to Jersey 3.1.5
```

Comment:
Would be good to:

1. Split into two entries.
2. Add some indication of a major breaking change (making the line bold and/or adding some emoji in the beginning like one of ❗️⚠️👉🚨‼️)
3. Move these to the top of the list.
4. Add entries under Removed:
   a. Removed support for Java 11.
   b. Removed support for Java EE and Jakarta EE 8.

Reply:
https://en.wikipedia.org/wiki/Jakarta_EE reference for the versions.

Reply:
Please see changes as suggested. ok?